test_utils/src/lib.rs

```diff
@@ -17,8 +17,8 @@ pub struct TestEnvironment {
 impl Drop for TestEnvironment {
     fn drop(&mut self) {
         match &self.original_home {
-            Some(home) => unsafe { env::set_var("HOME", home) },
-            None => unsafe { env::remove_var("HOME") },
```

Comment:
Why remove the unsafe here? Without the unsafe I get rust-analyzer warnings locally

Reply:
I only removed it because clippy was complaining about a useless unsafe.